### ISSUE TITLE
Can Add Mark and Space to Parity?

### ISSUE THREAD

**Issue by rrxmzl** [serialport/serialport-rs#167]
Like Qt
https://doc.qt.io/qt-6/qserialport.html#Parity-enum

QSerialPort::NoParity
QSerialPort::EvenParity
QSerialPort::OddParity
QSerialPort::SpaceParity
QSerialPort::MarkParity

**Comment by sirhcel** [comment]
Yes, this could be done. At least for a selection of target operating systems like Linux and Windows. Would you file a PR for it?
